drivers/platform/aducm3029/no_os_uart.c

```diff
@@ -291,7 +166,7 @@ int32_t no_os_uart_read(struct no_os_uart_desc *desc, uint8_t *data,
 
 	idx = 0;
 	while (bytes_number) {
-		to_read = no_os_min(bytes_number, NO_OS_MAX_BYTES);
+		to_read = no_os_min(bytes_number, NO_OS_UART_MAX_BYTES);
```

Comment:
The define needs to be moved to a wider namespace as part of the IRQ update and I wanted to make the name more specific to avoid potential name clashes. This is my argument for changing here. If we want we can make a separate commit for it. Let me know.